### DISCUSSION TITLE
When will i get the confirmation email?

### DISCUSSION THREAD

**Discussion by The-Panda-Plex-Cinema** [community/community#10489] [Copilot Conversations] [answered]
Hey,
When will i get the confirmation email?
I have been waiting for the confirmation email for a long time but i am not getting it.
Please tell me a rough time period I'll get it.
If any support staff is seeing this please check it asap.

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
